app/services/decisions/dwp_decision_tree.rb

```diff
@@ -6,13 +6,19 @@ def destination
         after_confirm_result
       when :confirm_details
         after_confirm_details
+      when :cannot_check_dwp_status
+        after_cannot_check_dwp_status
       else
         raise InvalidStep, "Invalid step '#{step_name}'"
       end
     end
 
     private
 
+    def after_cannot_check_dwp_status
+      edit('steps/client/has_benefit_evidence')
```

Comment:
@hiboabd What do you think of using  **:cannot_check_dwp_status** instead of string path

Reply:
The opposite problem for this one `has_benefit_evidence` is in the client scope 😩